<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Автоматизированное рабочее место</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(angle: 45deg, stop:0 #ee9ca7, stop:1 #ffdde1);
border-color: rgb(0, 0, 0);

</string>
   </property>
   <widget class="QGroupBox" name="refuelGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(90deg, #b9deed, #efefef);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>Заправка автомобиля</string>
    </property>
    <widget class="QLabel" name="fuelTypeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Тип топлива:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="fuelTypeComboBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>92</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>95</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DT</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="fuelAmountLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество топлива:</string>
     </property>
    </widget>
    <widget class="QLabel" name="paymentTypeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Тип оплаты:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="paymentTypeComboBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>Наличные</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Банковская карта</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Карта лояльности</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="refuelButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>131</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../build-ARM-Desktop_Qt_5_12_12_MinGW_32_bit-Debug/oil-can.png</normaloff>../build-ARM-Desktop_Qt_5_12_12_MinGW_32_bit-Debug/oil-can.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
    <widget class="QComboBox" name="workstationComboBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="workstationlabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Рабочее место:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="fuelAmountSpinBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="refuelButton_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>521</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(90deg, #b9deed, #efefef);</string>
    </property>
    <property name="title">
     <string>Настройка цен на топливо</string>
    </property>
    <widget class="QLabel" name="price92Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Цена за литр топлива 92:</string>
     </property>
    </widget>
    <widget class="QLabel" name="price95Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Цена за литр топлива 95:</string>
     </property>
    </widget>
    <widget class="QLabel" name="price100Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Цена за литр топлива 100:</string>
     </property>
    </widget>
    <widget class="QLabel" name="priceDTLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Цена за литр топлива ДТ:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pricesButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>121</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
text-align: center;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}

QPushButton:hover {
	background-color: rgb(255, 255, 0);
	color: rgb(0, 0, 0);
	
}</string>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="price92SpinBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="value">
      <double>45.500000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="price95SpinBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="value">
      <double>48.500000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="price100SpinBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="value">
      <double>52.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="priceDTSpinBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="value">
      <double>47.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="historyGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>1071</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(90deg, #b9deed, #efefef);</string>
    </property>
    <property name="title">
     <string>История заправок</string>
    </property>
    <widget class="QPushButton" name="historyButton">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
text-align: center;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}

QPushButton:hover {
	background-color: rgb(255, 255, 0);
	color: rgb(0, 0, 0);
	
}</string>
     </property>
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
    <widget class="QTableView" name="historyTableView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>1031</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(255, 255, 0);
color: rgb(255, 255, 255);

</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="resizeColumnsToContents" stdset="0">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="resizeButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
text-align: center;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}

QPushButton:hover {
	background-color: rgb(255, 255, 0);
	color: rgb(0, 0, 0);
	
}
</string>
    </property>
    <property name="text">
     <string>Сервисное меню</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
